_posts/2020-02-09-SVGD.markdown

```diff
@@ -0,0 +1,401 @@
+---
+layout: post
+title:  "Stein Variational Gradient Descent"
+date:   2020-02-09 6:00:00 -0400
+categories: steins-method, stein-variational-gradient-descent, bayesian-inference
+author: bhairav
+blurb: "Stein Variational Gradient Descent is a powerful, non-parameteric Bayesian Inference algorithm, based on a tool called Stein's Method. In this guide, we work through the mathematics and fundamentals to understand Stein's Method, working our way to Kernelized Stein Discrepancy and, finally, Stein Variational Gradient Descent. We take Reinforcement Learning as an application area, and view SVGD from the lens of gradient flow, bringing new insights and applications to the SVGD toolbox."
```

Comment:
It's a bit tough to parse on the main site. What do you tihnk of these edits?

"Stein Variational Gradient Descent is a powerful, non-parameteric Bayesian Inference algorithm. It is based on its namesake, Stein's Method. In this guide, we work through the mathematics and fundamentals behind this approach, including Kernelized Stein Discrepancy, before fully understanding Stein Variational Gradient Descent. We end by considering two application areas for SVGD. One is Reinforcement Learning, the other is viewing SVGD from the lens of gradient flow."